model/flow/sealing_segment.go

```diff
@@ -6,76 +6,34 @@ import (
 	"golang.org/x/exp/slices"
 )
 
-// SealingSegment is the chain segment such that the last block (greatest height)
-// is this snapshot's reference block and the first (least height) is the most
-// recently sealed block as of this snapshot (ie. the block referenced by LatestSeal).
+// SealingSegment is a continuous segment of recently finalized blocks that contains enough history
+// for a new node to execute its business logic normally. It is part of the data need to initialize
+// a new node to join the network.
+// DETAILED SPECIFICATION: ./sealing_segment.md
 //
-// In other words, the most recently incorporated seal as of the highest block
-// references the lowest block. The highest block does not need to contain this seal.
+//			├═══════════┤  ├───────────────────────┤
+//			 ExtraBlocks   ^          Blocks       ^
+//	                    B                      Head
```

Comment:
```suggestion
//   ├═══════════┤  ├───────────────────────┤
//    ExtraBlocks   ^          Blocks       ^
//                  B                      Head
```
was mixing tabs and spaces